Game/Src/UI/GloryUI.gd

```diff
@@ -1,22 +1,34 @@
 extends Control
 
-
 onready var full = $Full
 onready var filling = $Filling 
-var max_glory
+export var max_glory = 100
+
+signal glory_changed(value)
+
+var glory_unit
 
 func _ready():
-	max_glory = full.rect_scale.x 
-	print(max_glory)
+	glory_unit = full.rect_size.x / max_glory
+#	print('glory_unit ',glory_unit)
```

Comment:
delete

Reply:
The comment? SHould I do all these changes then?

Reply:
Not sure what you mean. I just saw a commented line and though it was not needed